rs/tests/BUILD.bazel

```
@@ -34,6 +34,7 @@ rust_library(
         "src/memory_safety.wasm",
         "src/counter.wat",
         "src/ledger_tests/transaction_ledger_correctness.wasm",
+        "src/cloner_canister.wasm.gz",
```

@basvandijk Can it still be included here as 
```suggestion
        "networking/cloner_canister.wasm.gz",
```

Or is there a better way to include it as compile data when building if it's only used in a networking test?

---

IIUC you're not using this at compile time (only at runtime) so just remove this.

---

Thanks

---

```
❯ ict test //rs/tests/networking:cloner_canister

WARNING: Streamed test output requested. All tests will be run locally, without sharding, one at a time
ERROR: /ic/rs/tests/networking/BUILD.bazel:234:12: no such target '//rs/tests:src/cloner_canister.wasm.gz': target 'src/cloner_canister.wasm.gz' not declared in package 'rs/tests' defined by /ic/rs/tests/BUILD.bazel (Tip: use `query "//rs/tests:*"` to see all the targets in that package) and referenced by '//rs/tests/networking:cloner_canister_workload'
ERROR: Analysis of target '//rs/tests/networking:cloner_canister_workload' failed; build aborted: Analysis failed
INFO: Elapsed time: 0.272s
INFO: 0 processes.
ERROR: Couldn't start the build. Unable to run tests
INFO: Streaming build results to: https://dash.idx.dfinity.network/invocation/f0755824-ff65-4818-8252-b57c3b46b073
FAILED: Build did NOT complete successfully (3 packages loaded, 6 targets configured)
There was an error while executing CLI: exit status 1
```

---

See [the following](https://github.com/dfinity/ic/pull/798/files#r1706907418):